README.md

```diff
@@ -0,0 +1,3 @@
+# repo from murphy update
+documentatuon
+now this is the second time i am going to add documentation to this file
```

Comment:
make the word file bold

Reply:
the word file is now in capital as requested